OpenRA.Mods.Common/Widgets/Logic/MainMenuLogic.cs

```diff
@@ -28,6 +28,8 @@ public class MainMenuLogic : ChromeLogic
 	{
 		protected enum MenuType { Main, Singleplayer, Extras, MapEditor, SystemInfoPrompt, None }
 
+		protected enum GameState { NONE, MISSIONS, SKIRMISH, MULTIPLAYER, MAP_EDITOR, REPLAYS }
```

Comment:
Style nit: Please use TitleCase for enum definitions, and rename this from `GameState` to `MenuPanel`.